drive-board-firmware/src/remote_control.c

```diff
@@ -95,6 +95,7 @@ static THD_FUNCTION(rc_thread_main, arg)
 
             // NOTICE("%f,%f", linear_x, angular_z);
             base_set_speed(&rover_base, linear_x, angular_z);
+            chThdSleepMilliseconds(10);
```

Comment:
This is weird, will investigate when there is time. until then, ship it!

Reply:
:shipit: